codice/ddf
# DDF-4235 Moving result form management out of workspace

#### What does this PR do?
Moves result form management outside the workspace. As well as removing result forms as an experimental feature.
Search form and result form icons were also updated
#### Who is reviewing it? 
@Bdthomson @alanchampion @hayleynorton 

#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@bdeining 
#### How should this be tested?
- Build/run
- Navigate to new result forms page
- Ensure no regression in result form functionality

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4235](https://codice.atlassian.net/browse/DDF-4235)
#### Screenshots
![resultform](https://user-images.githubusercontent.com/39923178/47118459-8792ca80-d224-11e8-9b17-59915b0b456c.gif)
<img width="1053" alt="screen shot 2018-10-29 at 2 04 05 pm" src="https://user-images.githubusercontent.com/39923178/47677013-acbbfd00-db83-11e8-8f8d-944889eaebf3.png">
<img width="992" alt="screen shot 2018-10-29 at 2 04 34 pm" src="https://user-images.githubusercontent.com/39923178/47677014-af1e5700-db83-11e8-842a-a3667808dbf0.png">



#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
